pkg/storaged/fsys-panel.jsx

```diff
@@ -62,53 +65,50 @@ export class FilesystemsPanel extends React.Component {
             for (var i = 0; i < mount_points.length && !fsys_size; i++)
                 fsys_size = client.fsys_sizes.data[mount_points[i]];
 
-            function go(event) {
-                if (!event || event.button !== 0)
-                    return;
-                go_to_block(client, path);
-            }
-
-            return (
-                <tr onClick={go} key={path}>
-                    <td>{ block.IdLabel || block_name(block) }</td>
-                    <td>
-                        { fsys.MountPoints.length > 0
+            return {
+                props: { path, client },
+                columns: [
+                    { title:  block.IdLabel || block_name(block) },
+                    {
+                        title: fsys.MountPoints.length > 0
                             ? fsys.MountPoints.map((mp) => <div key={mp}>{decode_filename(mp)}</div>)
                             : "-"
-                        }
-                    </td>
-                    <td>
-                        { fsys_size && fsys.MountPoints.length > 0
+                    },
+                    {
+                        title: fsys_size && fsys.MountPoints.length > 0
                             ? <StorageUsageBar stats={fsys_size} critical={0.95} />
                             : fmt_size(block.Size)
-                        }
-                    </td>
-                </tr>
-            );
+                    }
+                ]
+            };
         }
 
         var mounts = Object.keys(client.blocks).filter(is_mount)
                 .sort(cmp_mount)
```

Comment:
Do we still need to sort the rows explicitly?  Or does the table sort them anyway?

Reply:
AH good catch _ I can make the table have some initial sorted state without this.

Reply:
Fixed.